src/Raven.Server/ServerWide/Commands/GetRaftIndexCommand.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Raven.Server.ServerWide.Commands
+{
+    public class GetRaftIndexCommand : CommandBase
```

Comment:
Remember to remove this after: http://issues.hibernatingrhinos.com/issue/RavenDB-9597

Reply:
Wrote a reminder in the issue!!